wpa_supplicant/ctrl_iface_zephyr.h

```diff
@@ -16,15 +16,36 @@
 #include "ctrl_iface.h"
 #include "common/wpa_ctrl.h"
 
+/**
+ * struct wpa_ctrl_dst - Data structure of control interface monitors
+ *
+ * This structure is used to store information about registered control
+ * interface monitors into struct wpa_supplicant.
+ */
+struct wpa_ctrl_dst {
```

Comment:
`dst` might be misleading, we can use other term like `wpa_ctrl_mon` (IIRC, this is what WPAS uses as well)